iosHackthonApp/iosHackthonApp/Add Meal Screen/AddMealView.swift

```diff
@@ -8,48 +8,125 @@
 
-    @ObservedObject private(set) var viewModel: AddMealViewModel
-
-    var body: some View {
-        ZStack {}
-        .alert(isPresented: $viewModel.showingCollectionCode) {
-            Alert(
-                title: Text(viewModel.code),
-                message: Text(Strings.AddMealScreen.CollectionAlert.message),
-                dismissButton: .default(Text(Strings.Common.ok)))
-        }
-        ZStack {
-            VStack {
-            //TODO: Pass the actual values when UI is complete
-                Button("Add a meal") {
-                    let meal = Meal(
-                        id: "\(viewModel.sdk.getUUID())",
-                        name: "Pizza",
-                        quantity: 2,
-                        availableFromDate: "Tuesday",
-                        expiryDate: "Saturday",
-                        info: "Meat",
-                        hot: false,
-                        locationLat: 51.509865,
-                        locationLong:  -0.118092)
-                    self.viewModel.postMeal(meal: meal)
-                }
-            }
-        }
-        .alert(isPresented: $viewModel.showingError) {
-            Alert(
-                title: Text(Strings.Common.sorry),
-                message: Text(Strings.Common.ErrorAlert.message),
-                dismissButton: .default(Text(Strings.Common.ok)))
-        }
-    }
+	@ObservedObject private(set) var viewModel: AddMealViewModel
+	@State var title: String = ""
+	@State var additionalInfo: String = ""
+	@State var quantity: Int = 0
+	@State var availableFromDate = Date()
+	@State var useByDate = Date()
+	@State var address: String = ""
+	@State var latitude: Float = 0
+	@State var longitude: Float = 0
+	@State var isHot : Bool = true
+
+	var body: some View {
+
+	ZStack {}
+	.alert(isPresented: $viewModel.showingCollectionCode) {
+			Alert(
+					title: Text(viewModel.code),
+					message: Text(Strings.AddMealScreen.CollectionAlert.message),
+					dismissButton: .default(Text(Strings.Common.ok)))
+	}
+	ZStack {
+		
+		ScrollView {
+			VStack(alignment: .leading, spacing: 10) {
+
+				Group {
+					Text(Strings.AddMealScreen.title)
+					TextField("", text: $title)
+						.modifier(GrayTextFieldStyle())
+					Text(Strings.AddMealScreen.additionalInfo)
+					TextField("", text: $additionalInfo)
+						.modifier(GrayTextFieldStyle())
+					Text(Strings.AddMealScreen.quantity)
+					TextField("", value: $quantity, formatter: NumberFormatter())
+						.modifier(GrayTextFieldStyle())
+					Text(Strings.AddMealScreen.temperature)
+
+					HStack {
+						Button(action: {
+							isHot = true
+						}) {
+							let color = self.isHot == true ? ColorManager.red: ColorManager.gray
+							Image(systemName: Strings.AddMealScreen.Images.hotFood)
+								.modifier(IconButtonImageStyle(color: color))
+						}
+
+						Button(action: {
+							isHot = false
+						}) {
+							let color = self.isHot == false ? ColorManager.blue: ColorManager.gray
```

Comment:
and do `!self.isHot` or just `self.isHot` and switch the colours around

Reply:
Done